### ISSUE TITLE
The 'EFCorePowerToolsPackage' package did not load correctly.

### ISSUE THREAD

**Issue by adam-ciszewski** [ErikEJ/EFCorePowerTools#1089]
Hi, the tool was working great for me last couple weeks but today after launching VS I get this message.
No new extensions were installed since last time it worked and all I use other then this tool is ReSharper.

![image](https://user-images.githubusercontent.com/40968511/132122677-7ec2d9ff-6861-4c97-ad0c-6101b8d39686.png)

Getting this message both on load of VS and when trying to use the 'Reverse Engineering' option from context menu.

Here relevant content of the file mentioned in error popup.
```
  <entry>
    <record>835</record>
    <time>2021/09/05 09:48:37.425</time>
    <type>Information</type>
    <source>VisualStudio</source>
    <description>Begin package load [EFCorePowerToolsPackage]</description>
    <guid>{F4C4712C-CEAE-4803-8E52-0E2049D5DE9F}</guid>
  </entry>
  <entry>
    <record>836</record>
    <time>2021/09/05 09:48:37.545</time>
    <type>Error</type>
    <source>VisualStudio</source>
    <description>SetSite failed for package [EFCorePowerToolsPackage]Source: &apos;EFCorePowerTools&apos; Description: Could not load file or assembly &apos;Microsoft.VisualStudio.Threading, Version=16.10.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a&apos; or one of its dependencies. The system cannot find the file specified.&#x000D;&#x000A;System.IO.FileNotFoundException: Could not load file or assembly &apos;Microsoft.VisualStudio.Threading, Version=16.10.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a&apos; or one of its dependencies. The system cannot find the file specified.&#x000D;&#x000A;File name: &apos;Microsoft.VisualStudio.Threading, Version=16.10.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a&apos;&#x000D;&#x000A;   at EFCorePowerTools.EFCorePowerToolsPackage.InitializeAsync(CancellationToken cancellationToken, IProgress`1 progress)&#x000D;&#x000A;   at Microsoft.VisualStudio.Shell.AsyncPackage.&lt;&gt;c__DisplayClass20_0.&lt;&lt;Microsoft-VisualStudio-Shell-Interop-IAsyncLoadablePackageInitialize-Initialize&gt;b__1&gt;d.MoveNext()&#x000D;&#x000A;--- End of stack trace from previous location where exception was thrown ---&#x000D;&#x000A;   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)&#x000D;&#x000A;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#x000D;&#x000A;   at Microsoft.VisualStudio.Threading.JoinableTask.&lt;JoinAsync&gt;d__70.MoveNext()&#x000D;&#x000A;--- End of stack trace from previous location where exception was thrown ---&#x000D;&#x000A;   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()&#x000D;&#x000A;   at Microsoft.VisualStudio.Services.VsTask.RethrowException(AggregateException e)&#x000D;&#x000A;   at Microsoft.VisualStudio.Services.VsTask.InternalGetResult(Boolean ignoreUIThreadCheck)&#x000D;&#x000A;   at Microsoft.VisualStudio.Services.VsTask.GetResult()&#x000D;&#x000A;&#x000D;&#x000A;WRN: Assembly binding logging is turned OFF.&#x000D;&#x000A;To enable assembly bind failure logging, set the registry value [HKLM\Software\Microsoft\Fusion!EnableLog] (DWORD) to 1.&#x000D;&#x000A;Note: There is some performance penalty associated with assembly bind failure logging.&#x000D;&#x000A;To turn this feature off, remove the registry value [HKLM\Software\Microsoft\Fusion!EnableLog].&#x000D;&#x000A;</description>
    <guid>{F4C4712C-CEAE-4803-8E52-0E2049D5DE9F}</guid>
    <hr>80070002</hr>
    <errorinfo></errorinfo>
  </entry>
  <entry>
    <record>837</record>
    <time>2021/09/05 09:48:37.555</time>
    <type>Error</type>
    <source>VisualStudio</source>
    <description>End package load [EFCorePowerToolsPackage]</description>
    <guid>{F4C4712C-CEAE-4803-8E52-0E2049D5DE9F}</guid>
    <hr>80070002</hr>
    <errorinfo></errorinfo>
  </entry>
</activity>
```

### Further technical details
EF Core Power Tools version: 2.5.744

Database engine: SQL Server

Visual Studio version: Visual Studio 16.9.3

**Comment by ErikEJ** [comment]
Sorry, I updated my dependencies too aggressive. Update VS or get the latest daily.

**Comment by ErikEJ** [comment]
Sorry, I updated my dependencies too aggressive. Update VS or get the latest daily.

**Comment by ErikEJ** [comment]
I havejust published an update to MarketPlace to fix this for all VS 16.9 users
